src/java/org/apache/cassandra/service/paxos/Paxos.java

```diff
@@ -385,13 +386,18 @@ public EndpointsForToken readCandidates()
         }
 
         static Participants get(TableMetadata table, Token token, ConsistencyLevel consistencyForConsensus)
```

Comment:
I left this here as `Paxos` uses it... I updated the stuff that mattered for the existing tests to use the new API.

I did see `PaxosRepair` touches this as well, but I assume it didn't matter as we don't have data in paxos so that logic prob didn't get touched?